llvm/test/CodeGen/AMDGPU/phi-vgpr-input-moveimm.mir

```diff
@@ -32,38 +32,6 @@ body:             |
     S_BRANCH %bb.2
 ...
 
----
-# GCN-LABEL: name: phi_moveimm_subreg_input
```

Comment:
This test doesn't look legitimate to be invoked by a pass that needs SSA from. The presence of the subreg def makes this test a non-SSA one.

Reply:
So should I remove this testcase itself ? Also do we need to add this extracted testcase again to the original one ?

Reply:
Yes, avoid this new test case. I recommend deleting this instance from the original test file.

Reply:
Done